src/note_editor/NoteEditor_p.cpp

```diff
@@ -1706,7 +1706,7 @@ void NoteEditorPrivate::onRemoveResourceUndoRedoFinished(const QVariant & data,
 void NoteEditorPrivate::onRenameResourceDelegateFinished(QString oldResourceName, QString newResourceName,
                                                          Resource resource, bool performingUndo)
 {
-    QNDEBUG(QStringLiteral("NoteEditorPrivate::onRenameResourceDelegateFinished: old resource name = ") << oldResourceName
+    QNTRACE(QStringLiteral("NoteEditorPrivate::onRenameResourceDelegateFinished: old resource name = ") << oldResourceName
             << QStringLiteral(", new resource name = ") << newResourceName << QStringLiteral(", performing undo = ")
             << (performingUndo ? QStringLiteral("true") : QStringLiteral("false")) << QStringLiteral(", resource: ") << resource);
```

Comment:
I think here it was fine to have `QNDEBUG`:

1. Only a handful of data is printed here, no full note or resource info.
2. It is a slot so it could be useful to have a log entry whenever it is invoked, for troubleshooting purposes - so, it seems `QNDEBUG` is a better fit for this than `QNTRACE`.

Reply:
OK. but the resource is very huge. So I split the log to DEBUG with text and resource on TRACE.